bookstack/Dockerfile

```diff
@@ -47,6 +48,9 @@ RUN \
          /var/www/bookstack/tests \
          /var/www/bookstack/dev
 
+# PDF Rendering
+ENV WKHTMLTOPDF /usr/bin/wkhtmltopdf
```

Comment:
Why is this needed, we are installing a default OS package right?

Besides, please move it higher up the Docker file so we leverage the Docker layer caching better.

Reply:
It's not needed in the Docker file, it would be an Env var set for FPM, as per the others.